src/patternfly/components/TextInputGroup/examples/TextInputGroup.md

```diff
@@ -17,6 +17,17 @@ cssPrefix: pf-c-text-input-group
 {{/text-input-group}}
 ```
 
+### Disabled
+```hbs
+{{#> text-input-group text-input-group--id="disabled" text-input-group--modifier='pf-m-disabled'}}
+  {{#> text-input-group-main}}
+    {{#> text-input-group-text}}
+      {{> text-input-group-text-input aria-label="disabled text input group example" text-input-group-text-input--attribute='disabled' text-input-group--value='Disabled'}}
```

Comment:
* There isn't an `aria-label` arg on that hbs template but there should be.
* You'll also want to add `tabindex="-1"` to disable tabbing to it or anything in it.
* You could also do `text-input-group--IsDisabled` on the parent hbs template, and just have that set all the classes/attrs/labels you will always set if it's disabled. That makes it easier to use going forward.